### ISSUE TITLE
Favorite percentage higher than 100%

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#14543]
### Describe your problem!

A user reported this (btw: its my owned cache) to us:

The favorite percentage looks strange:
![image](https://github.com/cgeo/cgeo/assets/949669/9318b89a-c786-4d5d-b41d-8c7c308a54b2)

The cause for this is, that one finder (who awarded a favorite point) changed his found log into a "Write note" (don`t ask why...he does this all the time). This means the logcount is reduced by 1 while the favorite count remains the same.

### How to reproduce?

See GC-code above

### Actual result after these steps?

Favorite percentage >100%

### Expected result after these steps?

Lets just set a max value to 100% for such edge cases and we are done IMHO.

### Reproducible

Yes

### c:geo Version

2023.08.16

**Comment by Lineflyer** [comment]
I personally hope it will never drop below 100%...just kidding ;)
@cgeo/team If someone wants to play this cache feel invited to solve the start quest and contact me with solution. We are currently ramping up visitors and I need some "testers".

**Comment by baiti** [comment]
> I personally hope it will never drop below 100%...just kidding ;) @cgeo/team If someone wants to play this cache feel invited to solve the start quest and contact me with solution. We are currently ramping up visitors and I need some "testers".

I have spent some time "testing" and it didn't really take me long to come to the point where I could make a reservation.  How do I contact you with more information regarding the experience with the riddle?

**Comment by Lineflyer** [comment]
> ... will pursue your GC account ...

Yepp, thats the best way.

**Comment by ztNFny** [comment]
c:geo team event in @Lineflyer cache location? ;-)

**Comment by baiti** [comment]
Let me give a summary here: Even after only dealing with the riddle, I found it very cool and if I were close to the scene I'd want to go further and I am quite sure the percentage wouldn't drop below 100% afterwards ;)

**Comment by ztNFny** [comment]
The GC website also says 14 FP with 13 Founds, so imho >100% is the only correct output. Imho works as designed, wontfix.

**Comment by moving-bits** [comment]
> The GC website also says 14 FP with 13 Founds, so imho >100% is the only correct output. Imho works as designed, wontfix.

Ah, I had prepared a "fix" in parallel, see #14544.

Mathematically, values > 100% may be correct, but they are still confusing for the end user in this context, so maybe a "fix" is still to be preferred?

**Comment by ztNFny** [comment]
> > The GC website also says 14 FP with 13 Founds, so imho >100% is the only correct output. Imho works as designed, wontfix.
> 
> Ah, I had prepared a "fix" in parallel, see #14544.
> 
> Mathematically, values > 100% may be correct, but they are still confusing for the end user in this context, so maybe a "fix" is still to be preferred?

It's not up to the client to fix false input. Shit in shit out. If the problem was inside cgeo it should be fixed, but cgeo displays everything correctly. Thus my vote stays with wontfix.

**Comment by Lineflyer** [comment]
> It's not up to the client to fix false input. Shit in shit out. If the problem was inside cgeo it should be fixed, but cgeo displays everything correctly. Thus my vote stays with wontfix.

If we would take the percentage as presented on the website we would get 100%, but we decided to use calculate ourselves to save one additional request needed otherwise. So we don't handle that edge case while GC does.

BTW: Calculating ourselves also has the drawback, that the percentage differs from the website as soon as non-PM users log a PM-only cache: https://www.cgeo.org/faq#favpercentage
